src/v2/Apps/Artist/Routes/Works/index.tsx

```diff
@@ -22,20 +23,27 @@ export const WorksRoute: React.FC<WorksRouteProps> = props => {
     isClient &&
     get(artist, a => a.related.artistsConnection.edges.length, 0) > 0
 
+  const hasArtistSeries =
+    get(artist, a => a.artistSeriesConnection.edges.length, 0) > 0
```

Comment:
We can use the `?` operator here

Reply:
We use the `get` helper a few lines above for pretty much the same thing. It might be better to switch things all at once, vs. mixing and matching multiple patterns (I have a slight preference for not mixing multiple equivalent methods).